src/python/WMCore/WMRuntime/ProcessMonitor.py

```diff
@@ -146,8 +146,8 @@ def __init__(self):
         self.target = None
 
     def setTarget(self,newtarget):
-        if ((type(newtarget) != FunctionType) and
-            (type(newtarget) != LambdaType)):
+        if ((not isinstance(newtarget, FunctionType)) and
```

Comment:
Let me suggest something much cleaner here:
```suggestion
        if not isinstance(newtarget, (FunctionType, LambdaType)):
```

Reply:
I like this very much! Changed!